mailpoet/lib/Cron/Workers/SendingQueue/SendingQueue.php

```diff
@@ -128,7 +128,7 @@ public function process($timer = false) {
       $this->startProgress($task);
 
       try {
-        ScheduledTaskModel::touchAllByIds([$queue->taskId]);
```

Comment:
After this change, `ScheduledTaskModel` is only used for constants, can be replaced with `ScheduledTaskEntity`.